core/trino-main/src/main/java/io/trino/execution/SessionPropertyEvaluator.java

```diff
@@ -120,20 +123,20 @@ public static List<PropertyMetadata<?>> findSimilar(String propertyName, Set<Pro
     {
         return candidates.stream()
                 .filter(property -> !property.isHidden())
-                .map(candidate -> new Match(candidate, FuzzySearch.ratio(candidate.getName(), propertyName)))
-                .filter(match -> match.ratio() > 75)
-                .sorted(comparingInt(Match::ratio).reversed())
+                .map(candidate -> new Match(candidate, SIMILARITY.similarity(candidate.getName(), propertyName)))
+                .filter(match -> match.ratio() > 0.85)
```

Comment:
Is 0.85 (vs 75, previously) intentional or a typo?

Reply:
intentional as the algorithm is differnet